examples/mqtt/dcc_comms/aws_iot/sampleProp.conf

```diff
@@ -0,0 +1,26 @@
+#AWS IoT Configurations
+EdgeSystemName = "TestGatewayName"
+BrokerIP = "Broker Name or IP"
+BrokerPort = 8883
+keep_alive = 60
+ConnectDisconnectTimeout = 10
+cert_required = "CERT_REQUIRED"
+tls_version = "PROTOCOL_TLSv1_2"
+broker_root_ca_cert = "/etc/liota/mqtt/conf/ca.crt"
+broker_username = None
+broker_password = None
+edge_system_cert_file = "/etc/liota/mqtt/conf/client.crt"
```

Comment:
We didn't decide on location for cert files as they are user defined and path is obtained from `sampleProp.conf` and not from `liota.conf`.  However, for representation, it can be `/etc/liota/mqtt/`.

If a platform supports certification based authentication, DCCComms will need them.  If we're publishing to two different Dccs certificates might (not always) be different.